(feat): add deployment page (#350)
chalk-ai/docs: aws-cloud-deployment.mdx

import { ServiceDiagramSwitcher } from '@/components/ArchitectureDiagram'

> This page contains GCP specific information on deploying Chalk. A general

Reviewer: ```suggestion
This page contains AWS specific information on deploying Chalk. A general
```

> This page contains GCP specific information on Chalk deployment. A general

Author: ```suggestion
This page contains GCP specific information on deploying Chalk. A general
```

- `kms`: Encryption keys for secrets
- `secretsmanager`: Encrypting data source secrets from the web UI

## Granting the Roles

> The following bash script can be used to grant all the roles to the chalk provided service account:

Author: ```suggestion
The following bash script can be used to grant the necessary roles to the Chalk provided service account:
```

> The following bash script can be used to grant the necessary roles to the Chalk provided service account:

Author: ```suggestion
The following bash script can be used to grant the roles we need to deploy Chalk into your AWS project:
```

- `kms`: Encryption keys for secrets
- `secretsmanager`: Encrypting data source secrets from the web UI

## Granting the Roles

The following bash script can be used to grant all the roles to the chalk provided service account:

> 

Author: ```suggestion
```
